stories/components/Tooltip.stories.tsx

```diff
@@ -10,6 +10,9 @@ import type { ButtonProps } from '../../src/components/deprecated-button'
 
 export default {
     title: 'Components/Tooltip',
+    parameters: {
+        badges: ['accessible'],
```

Comment:
Does this one have jest-axe tests already?

Reply:
Nope, added.